pkg/commands/hosting_service/hosting_service_test.go

```diff
@@ -8,63 +8,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestGetRepoInfoFromURL(t *testing.T) {
```

Comment:
shame on me for testing a private function which then got refactored out of existence